src/resources/views/components/modals/follower.blade.php

```diff
@@ -0,0 +1,38 @@
+<div onclick="hidefollowerModal()" id="followerModal"
+    class="fixed left-0 top-0 bg-mydark bg-opacity-50 w-full h-full justify-center items-center opacity-0 hidden transition-opacity duration-500 z-50">
+    <div onclick="event.stopImmediatePropagation()"
+        class="bg-mycream rounded-lg shadow-md p-10 flex justify-center items-center">
+        <div id="followerModal" class="bg-mycream mx-auto overflow-hidden transition-opacity duration-500">
+            <div class="text-mydark font-bold text-center w-full mb-2">
+                <h1>People that followed you</h1>
+            </div>
+            <div class="rounded-lg my-2 p-6 bg-mycream h-96 overflow-scroll overflow-x-hidden">
+                <div class="flex flex-row p-2 w-auto">
+                    <div class="w-auto h-auto rounded-full ml-3">
+                        <img class="w-12 h-12 object-cover rounded-full shadow cursor-pointer"
+                            src="assets/images/moon.jpg" alt="moon">
+                    </div>
+
+                    <div class="flex flex-col my-2 ml-4 pr-12">
+                        <div class="text-mydark text-sm font-semibold cursor-pointer">
+                            Moon
+                        </div>
+
+                        <div class="text-mydark flex font-light text-xs">
+                            {{-- Follower count --}}
+                            33k followers
```

Comment:
is this supposed to be hard coded? or just a place holder. aren't the follower count be dynamic?